lisp/tmp/ess-package.el

```diff
@@ -0,0 +1,58 @@
+;;; ess-package.el --- user customization of ESS
```

Comment:
Can we move this into `etc`? Polluting lisp dir doesn't feel right to me. There will likely be  other helper scripts for our targets in the future.

Reply:
Shouldn't it go into `test/`?

Also, please use lexical binding in all new lisp files.

Reply:
Could be, but I think it could be used for other purposes than tests, like distro building. I guess that's the idea here.